moveit_ros/visualization/CMakeLists.txt

```diff
@@ -19,20 +19,21 @@ pkg_check_modules(OGRE OGRE)
 link_directories( ${OGRE_LIBRARY_DIRS} )
```

Comment:
I think this is indeed needed, as pkg-config only reports `-lOgreMain -lpthread` only.